docs/mobile_sdk.md

```diff
@@ -54,9 +65,8 @@ In previous example, you pass in the `myKey` and `myToken` you get from OpenWhis
 ```
 $ wsk property get --auth
 ```
-
 ```
-whisk auth              kkkkkkkk-kkkk-kkkk-kkkk-kkkkkkkkkkkk:tttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttt
+whisk auth        kkkkkkkk-kkkk-kkkk-kkkk-kkkkkkkkkkkk:tttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttt
```

Comment:
This is how we do it across the docs, as when we format in bluemix docs becomes two different blocks style differently one for terminal/pre and the other for output/screen